modules/sdk-core/src/bitgo/bip32util.ts

```diff
@@ -48,3 +48,10 @@ export function verifyMessage(
   );
   return bitcoinMessage.verify(message, address, signature, network.messagePrefix);
 }
+
+/**
+ Verifies if a message was signed using the given ecdh key
+ */
+export function verifyEcdhSignature(message: string, signature: string, ecdhXpub: string): boolean {
+  return bip32.fromBase58(ecdhXpub).verify(Buffer.from(message), Buffer.from(signature, 'hex'));
+}
```

Comment:
Please create an isolated unit test just for this (not just as part of the integration tests).